libcontainer/intelrdt/intelrdt.go

```diff
@@ -330,7 +330,16 @@ func writeFile(dir, file, data string) error {
 	if dir == "" {
 		return fmt.Errorf("no such directory for %s", file)
 	}
-	if err := os.WriteFile(filepath.Join(dir, file), []byte(data+"\n"), 0o600); err != nil {
+
+	f, err := os.OpenFile(filepath.Join(dir, file), os.O_WRONLY|os.O_APPEND, 0o600)
+	if err != nil {
+		return fmt.Errorf("failed to open file %s: %w", file, err)
```

Comment:
nit: no need to wrap `err` here, as `os.OpenFile` already returns a `PathError`, which, when converted to a string, says "open <filename>: <strerror>"

Reply:
Thx, fixed